bloom-instance/main.tf

```diff
@@ -19,13 +19,16 @@ provider "aws" {
 }
 
 locals {
-  default_name = "${var.resource_prefix}-${var.sdlc_stage}-${var.application_name}"
+  # We may want to rearrange the order of these in the future based on how easy
+  # or hard it is to read at a glance in the console.
+  default_name = "${var.resource_prefix}:${terraform.workspace}-${var.sdlc_stage}:${var.application_name}"
```

Comment:
@chriscasto can you add a test plan screenshot? we should be able to see the output of `terraform plan` change I think?

Reply:
(also I approve of the `-` to `:` switch)

Reply:
You got it

![Screenshot 2023-02-21 at 4 47 29 PM](https://user-images.githubusercontent.com/23032560/220484775-49e77339-cace-4e3a-a5b2-d4609a315360.png)